modules/cnf-programming-the-n3000-operator.adoc

```diff
@@ -0,0 +1,351 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="programming-the-openness-operator-for-intel-fpga-pac-n3000-operator_{context}"]
```

Comment:
Like previous comment about the ID, it's cheap to make it match the file name exactly.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata